specification/attributes/unit_format.md

```diff
@@ -0,0 +1,92 @@
+# Unit Format
+
+Cost and utilization data frequently captures data measured in units related to data size, count, time, and other dimensions. The Unit Format attribute provides a standard for expressing units of measure in columns appearing in billing data.
+
+All columns defined in FOCUS specifying Unit Format as a value format MUST follow the requirements listed below.
```

Comment:
I'm totally fine with this caveat, but if you prefer to not have it, we could also just call out PricingMeasurementUnit as an exception to this rule with the understanding that we explicitly want the provider's value and "unit" is the most logical name.

Reply:
I think this is the pattern we should follow for all columns that are governed by an attribute TBH. That way we have a very clear way of saying in the content constraints, if the value format is X, this column must meet requirements specified in X for al it's values.